portfolio/src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Message {
+  
+  private final long timestamp;
+  private final String message;
+
+  public Message(long timestamp, String message) {
+    this.timestamp = timestamp;
+    this.message = message;
+  }
+
+  public boolean isValid() {
+    if (message.trim() == "") {

```

Comment:
is it possible that message variable is null?

Reply:
Not from the form but might be if the request is modified, added a check for null!